src/applications/vaos/appointment-list/components/RequestedStatusAlert.jsx

```diff
@@ -11,27 +11,23 @@ import {
 } from '../../utils/constants';
 import { startNewAppointmentFlow } from '../redux/actions';
 
-function handleViewClick() {
```

Comment:
Just curious why this function was deleted and moved inline.

Reply:
@vbahinwillit Took a look at this. I believe the reasoning was that because it was only doing one thing (firing off an analytics event) that it should be inline. I also looked for another instance of `handleView()` in case that was a pattern.

Happy to put it back if I'm breaking a convention that we've been using of course, just let me know.

Reply:
No. Just curious.